src/main/java/Ejercicio6/AumentarHP.java

```diff
@@ -0,0 +1,15 @@
+package Ejercicio6;
+
+public class AumentarHP {
+    public static Pokemon[] aumentarHPPokemon(Pokemon[] pokemonArray) {
+        var nuevoArreglo = new Pokemon[pokemonArray.length];
+        var i = 0;
+        for (var elemento : pokemonArray) {
+            elemento.setHp(elemento.getHp() + 500);
+            nuevoArreglo[i] = elemento;
+            i++;
+
+        }
```

Comment:
usa un for convencional for(var i=0; i < n; i++)

Reply:
Corregido